### PR TITLE
BUG: No permission to chmod when multi users on the same machine

### PR DESCRIPTION
**Pull request by ChengjieLi28**
## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass

### CODE REVIEW

File: python/xorbits/_mars/constants.py [hunk 1]
```diff
@@ -14,8 +14,10 @@
 
 import os
 
-MARS_TEMP_DIR = "/tmp/xorbits"
-MARS_TEMP_DIR_WIN = "C:\\Temp\\xorbits"
+from .utils import get_current_user
```
Comment (qianduoduo0904) [inline python/xorbits/_mars/constants.py:17]: Xoscar 0.0.4 released, you can try to use the identical function from it.

Comment (ChengjieLi28) [reply python/xorbits/_mars/constants.py]: Fix

File: python/xorbits/_mars/constants.py [hunk 2]
```diff
@@ -13,18 +13,16 @@
 # limitations under the License.
 
 import os
+from pathlib import Path
 
-MARS_TEMP_DIR = "/tmp/xorbits"
-MARS_TEMP_DIR_WIN = "C:\\Temp\\xorbits"
+MARS_TEMP_DIR = f"{str(Path.home())}/.xorbits"
```
Comment (qianduoduo0904) [inline python/xorbits/_mars/constants.py:18]:
```suggestion
MARS_TEMP_DIR = str(Path.home() / '.xorbits')
```

**Review by qianduoduo0904 — APPROVED** [review]
LGTM

File: python/xorbits/_mars/constants.py [hunk 3]
```diff
@@ -13,18 +13,16 @@
 # limitations under the License.
 
 import os
+from pathlib import Path
 
-MARS_TEMP_DIR = "/tmp/xorbits"
-MARS_TEMP_DIR_WIN = "C:\\Temp\\xorbits"
+MARS_TEMP_DIR = str(Path.home() / ".xorbits")
```
[the remark this answers is not in the document]
Comment (qianduoduo0904) [reply python/xorbits/_mars/constants.py:18]: Absolutely, Path.home() returns PosixPath on unix-like system, and WindowsPath on windows.

[the remark this answers is not in the document]
Comment (ChengjieLi28) [reply python/xorbits/_mars/constants.py:18]: I  have tried this on Windows. This is a hidden dir on Windows and can be created successfully by os.makedirs.

**Review by UranusSeven — APPROVED** [review]
LGTM

**Pull request merged**